src/JavaDocs/DateTime.html

```diff
@@ -0,0 +1,324 @@
+<!DOCTYPE HTML>
```

Comment:
There's a bunch of html files that I don't really understand whats going on. Could you kindly teach me if I am missing out anything? Why do you have all these html files in your duke project?

Reply:
Hmm all these html files are the html version of the JavaDocs generated. If you use IntelliJ, you can generate JavaDocs by going to Tools, and then click on "generate JavaDocs", it will open the html file in the browser and save it to your repo.